src/components/ColorPicker/ColorPickerItem/ColorPickerItem.st.css

```diff
@@ -0,0 +1,61 @@
+:import {
+    -st-from: "../../../common/formatters.st";
+    -st-named: applyUnit, color, fallback, value, applyOpacity, zeroAsTrue;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: RadioButton;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultItemSize: 24;
+    DefaultBorderRadius: 12;
+    DefaultFocusColor: color-5;
+}
+
+:vars {
+    /*Overrides*/
+    Size: --overridable;
+    Radius: --overridable;
+}
+
+:vars {
+    colorPickerItemSize: applyUnit(fallback(value(Size), value(DefaultItemSize)), px);
+    colorPickerBorderRadius: applyUnit(fallback(zeroAsTrue(value(Radius)), value(DefaultBorderRadius)), px);
```

Comment:
I think we shouldn't apply the `px` unit here.
Because TPA's might want to use percent.

Reply:
Ping 👆

Reply:
done